spec/lib/adviser_qualifications_lookup_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe AdviserQualificationsLookup do
+  let(:adviser) { FactoryGirl.create(:adviser, adviser_qualifications) }
+
+  let(:qualifications) { Array.new(3) { FactoryGirl.create(:qualification) } }
+
+  let(:adviser_qualifications) do
+    {
+      qualifications: [qualifications.first, qualifications.last]
+    }
+  end
+
+  let(:subject) { described_class.new([adviser.id]) }
+
+  describe '#for' do
+    it 'returns the qualifications for an advisor' do
+      expect(subject.for(adviser.id)).to eq(
+        [qualifications.first.name, qualifications.last.name]
```

Comment:
It would be better for the `eq` clause to contain the actual expected values

Reply:
👍 Ok, will change this.

Reply:
@margOnline because the qualification & accreditation factories generate names based on ids, eg. `Qualification 1` it's tricky to test those values reliably when the ids will change. I could get around it by passing a specific name in when we create the qualifications/accreditations at the top of the specs - would that be preferable?